src/main/java/org/jenkinsci/plugins/GithubSecurityRealm.java

```diff
@@ -106,7 +106,7 @@ of this software and associated documentation files (the "Software"), to deal
     private static final String DEFAULT_WEB_URI = "https://github.com";
     private static final String DEFAULT_API_URI = "https://api.github.com";
     private static final String DEFAULT_ENTERPRISE_API_SUFFIX = "/api/v3";
-    private static final String DEFAULT_OAUTH_SCOPES = "read:org,user:email";
+    private static final String DEFAULT_OAUTH_SCOPES = "read:org,user:email,repo";
```

Comment:
I'm OK with this.  It makes sense to update the scopes considering the GitHub committer authorization strategy.  The original scopes were meant just for the Project Matrix Authorization Strategy and didn't really account for all features of the plugin.

I'll have to update the wiki to document this when this change is merged and a release is published.